packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -44,6 +44,12 @@ export namespace FileNavigatorCommands {
         id: 'navigator.toggle.hidden.files',
         label: 'Toggle Hidden Files'
     };
+    export const REFRESH_NAVIGATOR: Command = {
+        id: 'navigator.refresh',
```

Comment:
@akosyakov just a question, should we start using the same command ids present in VSCode so plugins can reference these commands? Else from my understanding we will need to handle it as part of https://github.com/theia-ide/theia/pull/5590?

Reply:
We can do it in the plugin system as an adapter layer to VS Code. It would be nice to keep flexibility to do differently in core and do as VS Code in plugin system.

Reply:
Sounds good, thanks for clarifying :)